sashimi-webapp/src/logic/documentPackager/markdownProcessor.js

```diff
@@ -0,0 +1,41 @@
+// Imports:
+// Markdown-It module for markdown processing
+import MarkdownIt from 'markdown-it';
+// Code highlighting plugins
+import hljs from 'highlight.js'; // https://highlightjs.org/
+import mh from 'markdown-it-highlightjs';
+// Importing markdown-it-katex plugin for parsing LaTeX mathematical forumla
+import mk from 'markdown-it-katex';
+
+const md = new MarkdownIt({
+  html: true,
+  breaks: true,
+  linkify: true,
+  typographer: true,
+  highlight(str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        const val = hljs.highlight(lang, str, true).value;
+        return `<pre class="hljs"><code>${val}</code></pre>`;
+      } catch (__) { return ''; }
```

Comment:
Agree, should have named `__` with `error` instead.
But I guess in this case, it is also better to explain what this code is trying to do.